docker-compose.dev.yml

```diff
@@ -8,14 +8,14 @@ services:
       context: ./client
       dockerfile: Dockerfile.dev
     ports:
-    - 4000:4000
+      - 4000:4000
```

Comment:
Nitpick: We should decide on a format for YAML and stick with it. If we like our arrays indented, we may want to align the OpenShift templates with this format.

Reply:
I personally like indentation, just feels easier to read to me. But I'm good to go with whatever format, consistency is more important to me than having it one way or the other

Reply:
I just realized this now, this is what my VSCode do when I format yaml files.

Reply:
Decided as a team we'll start getting these indented going forward. Requires no updates to this PR.